src/events.rs

```diff
@@ -55,6 +55,8 @@ macro_rules! make_event {
         }
 
         impl $name {
+            #[inline] pub fn current_target(&self) -> Option<EventTarget> { self.event.current_target() }
```

Comment:
Why do you need `current_target`? Isn't `with_node` strictly better in every way?

Reply:
You're right! I used it when I did not yet know about `with_node`. That's a more elegant solution. I removed this part again.